test/tests/run-bash-in-container.sh

```diff
@@ -4,4 +4,4 @@ set -e
 testDir="$(readlink -f "$(dirname "$BASH_SOURCE")")"
 runDir="$(dirname "$(readlink -f "$BASH_SOURCE")")"
 
-source "$runDir/run-in-container.sh" "$testDir" "$1" bash ./container.sh
+source "$runDir/run-in-container.sh" "$testDir" "$1" sh ./container.sh
```

Comment:
This should instead be a new `run-sh-in-container.sh` script which is then symlinked into `test/tests/python-hy/run.sh` (where `run-bash-in-container.sh` is symlinked now).